lesson09/index.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+const ul = document.getElementById("ul");
+
+function createListDom(itemList) {
+    const frag = document.createDocumentFragment();
+
+    itemList.forEach(function(item) {
+        const li = document.createElement("li");
+        const anchor = document.createElement("a");
+        const image = document.createElement("img");
+        anchor.href= `/${item.to}`;
+        anchor.textContent = item.text;
+        image.src = item.img;
+        image.alt = item.alt;
+        frag.appendChild(li).appendChild(anchor).appendChild(image)
+    });
+
+    return frag;
+}
+
+function createLoading() {
+    const loadingImgWrap = document.createElement("div");
+    const image = document.createElement("img");
+    loadingImgWrap.classList.add("js-loading");
+    image.src = "./img/loading-circle.gif";
+    image.classList.add("loading-img");
+    loadingImgWrap.appendChild(image);
+    ul.before(loadingImgWrap);
+}
+
+function removeLoading() {
+    const loadingImage = document.querySelector(".js-loading");
```

Comment:
IMO

You get element by ID better than by class.
I think class is used for CSS.

Reply:
Thank you for your advice!
I rewritten to ID

Reply:
IMO 

This is also OK. But,
you should probably use class for CSS.
To separate of  roles of JS and CSS.

Reply:
Thank you for your advice!
I added class for css.